### PR TITLE
Contribution Guide CLI

### PR DESCRIPTION
**Pull request by Malyuk-A**
NOTE: This PR is built on top of the CLI Manuals branch - which is not yet merged - so the vast majority of changes are part of the other PR. - The only relevant file is `content/docs/contribution-guide/cli.md`

This PR is a rather minimal page explaining how to contribute to the CLI

### CODE REVIEW

File: content/docs/contribution-guide/cli.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  noindex: false # false (default) or true
+---
+
+## Report a Bug or Request a Feature
+
+Go to the CLI Repository and create a new Issue.
+
+{{< link-card
+  title="CLI Repository"
+  description="Explore the Source Code"
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: "Source Code" -> "source code"

File: content/docs/contribution-guide/cli.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  href="https://github.com/oakestra/oakestra-cli"
+  target="_blank"
+>}}
+
+
+## CLI Foundations
+The `oak-cli` is built via [Poetry](https://python-poetry.org/) and [Typer](https://typer.tiangolo.com/).<br>
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: Remove `<br>` tags here.

File: content/docs/contribution-guide/cli.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+We configured an automatic CI (GitHub Actions) to build and release these changes.<br>
+This CI will be triggered by a new tag.<br>
+```bash {title="Example Commands"}
+git tag -a vX.Y.Z -m "<New Custom Tag Message>"
+git push origin vX.Y.Z
+```
+---
+
+{{< callout context="tip" title="Did you know?" icon="outline/seeding" >}}
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: I would remove this for now as I want to have a separate contributor section with everyone who contributed and to whcih sections of Oakestra they helped. The docs can be free of dev names

File: content/docs/contribution-guide/cli.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+title: "CLI"
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: Command Line Interface (CLI)

**Review by nitindermohan — CHANGES_REQUESTED** [review]
Minor description changes required

File: content/docs/contribution-guide/cli.md (hunk 3 above)
Comment (Malyuk-A) [reply content/docs/contribution-guide/cli.md]: I agree

**Review by nitindermohan — APPROVED** [review]
Looks good now. Thanks Alex!

**Comment by nitindermohan** [comment]
@Malyuk-A there are merge conflicts to take care of. Once those are done, I wil approve the merge

File: content/docs/contribution-guide/cli.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Philosophy
+The `oak-cli` is a gateway to Oakestra and a multifaceted set of tools.
+Ensure high cohesion and low coupling by splitting unrelated/different parts into their own files and ["typer apps"](https://typer.tiangolo.com/tutorial/subcommands/add-typer/).
+Use the static CLI configuration, local machine purposes, and their filters.
+Users should always have access to generic/universal features - specialized features should only be available in fitting conditions/use-cases/environments.
+
+## Local Development
+For local development clone the CLI repository and install the CLI.
+```bash {title="Example Command"}
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: Remove title

```suggestion
```bash
```

File: content/docs/contribution-guide/cli.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+pip install -e .
+```
+Create a new branch and create a Pull Request as usual.
+Note that the final/merged CLI changes always require a CLI version increase that has to be followed up by a matching tag.
+
+We configured an automatic CI (GitHub Actions) to build and release these changes.
+This CI will be triggered by a new tag.
+```bash {title="Example Commands"}
```
Comment (nitindermohan) [inline content/docs/contribution-guide/cli.md]: Remove title

```suggestion
```bash
```

**Comment by nitindermohan** [comment]
@Malyuk-A very minor change suggestions. Can you please see if you want to include it. I will wait for your ACK before merging

**Comment by Malyuk-A** [comment]
Ok - I will get to this in ~1h

**Review by nitindermohan — CHANGES_REQUESTED** [review]

**Pull request merged**